solarforecastarbiter/validation/validator.py

```diff
@@ -660,3 +661,41 @@ def detect_clipping(ac_power, window=4, fraction_in_window=0.75, rtol=5e-3,
         flags = flags | _label_clipping(temp, window=window,
                                         frac=fraction_in_window)
     return flags
+
+
+def detect_clearsky(ghi, ghi_clearsky):
+    """ Identifies times when GHI is consistent with clear sky conditions.
```

Comment:
probably out of scope for this PR, but would be nice to eventually support clear sky detection for power and DNI. If this function is going to be specific to GHI then maybe `detect_clearsky_ghi`

Reply:
Yes, out of scope. Agree it would be nice to have the other functions, but I don't know of methods to do that. Agree with the name.

Reply:
Let's make some follow up issues for power and longer intervals.